coordinator/Dockerfile

```diff
@@ -1,20 +1,47 @@
-FROM openjdk:22-ea-17-slim-bookworm
+FROM openjdk:21-jdk-slim AS builder
 
-RUN apt-get update \
-	&& apt-get install curl -y \
-  && apt-get install unzip -y \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install unzip -y
+
+RUN mkdir -p /src /libs
+
+COPY --from=zip ./coordinator.zip /src
+
+RUN unzip /src/coordinator.zip -d /libs
+
+RUN mkdir -p /libs/coordinator/lib/unpacked-blob-compressor /libs/coordinator/lib/unpacked-blob-shnarf-calculator && \
+  cd /libs/coordinator/lib/unpacked-blob-compressor/ && jar -xf ../blob-compressor-0.0.4.jar && \
+  cd /libs/coordinator/lib/unpacked-blob-shnarf-calculator/ && jar -xf /libs/coordinator/lib/blob-shnarf-calculator-0.0.4.jar
+
+RUN case $(uname -m) in \
+    x86_64) \
+      rm -rf /libs/coordinator/lib/unpacked-blob-compressor/darwin-**; \
```

Comment:
Can this be moved out from the switch case here? seems they would be removed regardless of the case